lib/ffmpeg/movie.rb

```diff
@@ -13,13 +13,15 @@ class Movie
 
     UNSUPPORTED_CODEC_PATTERN = /^Unsupported codec with id (\d+) for input stream (\d+)$/
 
-    def initialize(path)
+    def initialize(path, skip_head_request: true)
```

Comment:
@aaeabdo did you try to use `false` as a default value? Does it require much changes in other places?

Reply:
I just think that instead of changing default behaviour it'd be safer and more of `as expected` to introduce option which is being changed only when needed